src/ops.jl

```diff
@@ -6,20 +6,40 @@ const tf = TensorFlow # so know where op_funcs is defined
 using MacroTools
 
 if VERSION < v"0.6.0-"
-    import Base: .*, .+, ./, .-, .^, .==
+    import Base: .*, .+, ./, .-, .^, .==, .!=
 end
 
-
-
-function tf_promote(t, x::Number)
```

Comment:
Are you sure we don't want this?
This is a surprisingly breaking change.
Because it is what makes writing `- 1` work on `Tensor{Int32}`s to get back an `Tensor{Int32}` and not a `Tensor{Int64}`

This does being us inline with the change that was made from 0.5 to 0.6
https://discourse.julialang.org/t/is-this-expected-behavior-0-5-to-0-6-breakage/2534/7

This explains the extra convert calls I noted in in the Imported_ops generation code.